specifyweb/businessrules/views.py

```diff
@@ -1 +1,284 @@
+                                "required": ["fields", "scope"]
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "responses": {
+            "201": {
+                "description": "Uniqueness rules checked for validation",
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "totalDuplicates": {
+                                    "type": "number",
+                                    "minimum": 0
+                                },
+                                "fields": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "object",
+                                        "properties": {
+                                            "duplicates": {
+                                                "type": "number",
+                                                "minimum": 0
+                                            },
+                                            "fields": {
+                                                "type": "array",
+                                                "items": {
+                                                    "type": "object",
+                                                    "examples": {
+                                                        "catalognumber": "012345678"
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "additionalProperties": False
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }})
+@require_POST
+def validate_uniqueness(request):
+    data = json.loads(request.body)
+    table = datamodel.get_table(data['table'])
+    django_model = getattr(models, table.django_name, None)
+
+    if table is None or django_model is None:
+        return http.HttpResponseBadRequest('Invalid table name in request')
+
+    uniqueness_rule = data['rule']
+    fields = [field.lower() for field in uniqueness_rule['fields']]
+    scope = uniqueness_rule['scope'].lower(
+    ) if uniqueness_rule['scope'] is not None else None
+
+    required_fields = {field: table.get_field(
+        field).required for field in fields}
+
+    strict_search = data["strict"] if 'strict' in data.keys() else False
+
+    strict_filters = Q()
+    for field, is_required in required_fields.items():
+        if not strict_search and not is_required:
+            strict_filters &= (~Q(**{f"{field}": None}))
```

Comment:
Not an immediate use case, but I left it as an option in case those using the API required that functionality. 
I can not imagine a case where `strict=True` would be used, so this can probably be removed if the extra code lends itself to unnecessary complication and bloat.